suites/apis/requestorNewTest.js

```diff
@@ -11,15 +11,35 @@ Feature('RequestorNewAPI @requires-requestorNew');
 const { expect } = require('chai');
 const requestorTasks = require('../../services/apis/requestor/requestorTasks.js');
 
+BeforeSuite(async ({
+  I,
+}) => {
+  I.cache = {};
+  I.cache.requestDidList = [];
+});
+
 Before(async ({
   home, users, login,
 }) => {
   home.do.goToHomepage();
   login.complete.login(users.mainAcct);
 });
 
+AfterSuite(async ({ I }) => {
+  console.log('Deleting all the requests created by this test with user cdis.autotest@gmail.com...');
+  if (process.env.DEBUG === 'true') {
+    console.log(I.cache.requestDidList);
+  }
+  I.cache.requestDidList.forEach(async (request) => {
+    const deleteRequest = await requestorTasks.deleteRequest(request);
+    if (deleteRequest === undefined) {
```

Comment:
Just curious, what would be the result of the test, when the test fails?

Reply:
so I have added a return to `deleteRequest` function. So now we look at the status of the deletion and say the request id is deleted.

Reply:
I think this only checks that request was triggered which will be true mostly, we should check the status code of the response or something